ADOPTERS.md

```diff
@@ -16,6 +16,18 @@ Details:
 - 760000 samples/s
 - 35M active series
 
+## Kinvolk Lokomotive Kubernetes
+
+https://kinvolk.io/lokomotive-kubernetes/
+
+Environments: AKS, AWS, Bare Metal, Equinix Metal
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes
+
+Details:
+- Lokomotive is self-hosted, security-centric and composable Kubernetes distribution for bare-metal and cloud platforms
```

Comment:
Maybe a bit too much of a pitch at least how I read it, I understand you didn't mean that btw! :) Maybe just include actual technical details of how and where you deploy or how many users use it would be a bit better? See OpenShift or GiantSwarm section for inspiration :) Thanks!